### PR TITLE
Add UUIDConverter

### PR DESCRIPTION
**Pull request by BalusC**
https://github.com/jakartaee/faces/issues/1819

### CODE REVIEW

**Comment by hantsy** [comment]
Add Chinese translation https://github.com/eclipse-ee4j/mojarra/pull/5244

**Comment by pizzi80** [comment]
when will be merged?

**Comment by BalusC** [comment]
> when will be merged?

When one reviewer has approved it.

File: impl/src/main/java/jakarta/faces/convert/UUIDConverter.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        }
+
+        if (value == null) {
+            return "";
+        }
+
+        try {
+            return value.toString();
+        } catch (Exception e) {
+            throw new ConverterException(MessageFactory.getMessage(context, STRING_ID, value, MessageFactory.getLabel(context, component)), e);
```
Comment (pzygielo) [inline impl/src/main/java/jakarta/faces/convert/UUIDConverter.java:108]: Just out of curiosity - assuming `value.toString()` throws - what is expected to happen with the message [for `ConverterException`] preparation? I suppose another `value.toString` call to be made...

Comment (BalusC) [reply impl/src/main/java/jakarta/faces/convert/UUIDConverter.java:108]: Sharp. This logic was just extracted as a template from existing faces converters.

**Review by arjantijms — APPROVED** [review]

**Pull request merged**